src/api/user/StatusAPI.js

```diff
@@ -100,6 +102,62 @@ export default class StatusAPI extends EventEmitter {
     }
   }
 
+  /**
+   * Can the currently logged in user set the mission status.
+   * @returns {Promise<Boolean>} true if the currently logged in user can set the mission status, false otherwise.
+   */
+  canSetMissionStatus() {
+    const provider = this.#userAPI.getProvider();
+
+    if (provider.canSetMissionStatus) {
+      return provider.canSetMissionStatus();
+    } else {
+      return Promise.resolve(false);
+    }
+  }
+
+  getStatusForMissionRole(role) {
```

Comment:
So as part of "Mission Status" we are setting GO or NO GO for some "categories":

- Commanding
- Imagery
- Driving

With the nomenclature I'm using, a "Status" would be GO or NO-GO (or something else). I then need something to call the categories, so I landed on "Mission Roles". Open to any other suggestions though. I think with proper documentation it can be clear